### DISCUSSION TITLE
Transfer Request for Valkyrie to typst-community

### DISCUSSION THREAD

**Discussion by jamesrswift** [typst-community/org#8] [Transfer Requests]
### Project Name

Valkyrie

### Initial Author(s)

James Swift

### Repository URL

https://github.com/JamesxX/valkyrie/

### Status

Looking for volunteers

### Description

This package implements type validation, and is targetted mainly at package and template developers. The desired outcome is that it becomes easier for the programmer to quickly put a package together without spending a long time on type safety, but also to make the usage of those packages by end-users less painful by generating useful error messages

### License

GNU General Public License v3.0

**Comment by tingerrr** [comment]
I have contacted @jcbhmr on the Typst community discord, but there was no answer yet. I mainly did so because I don't know what the process of transferring a package should be.

Since I have access to creating repositories on the org, I can do the transfer too.

**Comment by huwaireb** [comment]
An invitation was extended over to you James, looks like you've joined. You should be able to transfer the repository over now.
Once the transfer is complete, i'll close this request as completed.
